src/test/regress/pg_regress_multi.pl

```diff
@@ -90,6 +90,7 @@ ()
 push(@pgOptions, '-c', "unix_socket_directories=");
 push(@pgOptions, '-c', "fsync=off");
 push(@pgOptions, '-c', "shared_preload_libraries=citus");
+push(@pgOptions, '-c', "max_prepared_transactions=1000");
```

Comment:
Minor, 100 probably makes more sense, since default max_connections is 100 (you could never have more).